package/kedro_viz/launchers/jupyter.py

```diff
@@ -175,7 +116,7 @@ def run_viz(port: int = None, local_ns: Dict[str, Any] = None) -> None:
     viz_process.start()
     _VIZ_PROCESSES[port] = viz_process
 
-    _wait_for(func=_check_viz_up, host=host, port=port)
+    wait_for(func=check_viz_up, host=host, port=port)
```

Comment:
What is the reason to change from private function to public one?

Reply:
Earlier these functions were inside jupyter.py and used only by jupyter, now it is also used in cli launcher. so moved it to a utils file and these functions are now used in both places

Reply:
I see they are now shared among two modules, but I think it should be an internal method `_wait_for` instead of `wait_for`.

For example, we have similar internal utils here:
https://github.com/kedro-org/kedro/blob/edbbcc7f7a863b7c980f9efcaa444e2fa01f45fa/kedro/config/common.py#L209

Changing from `_wait_for` -> `wait_for` singals this is a public function. (though in reality 99% people are using kedro-viz as an appllication but not a module). So this is just to keep things clean internally for ourselves.

https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-single-and-double-underscore-before-an-object-name